lib/base/application.cpp

```diff
@@ -180,7 +180,7 @@ void Application::SetResourceLimits(void)
 	rlimit rl;
 
 #	ifdef RLIMIT_NOFILE
-	rl.rlim_cur = 16 * 1024;
+	rl.rlim_cur = GetRLimitFiles();
```

Comment:
We should allow a user to set to values to `NULL`, and then not set limits at all.

For all 3 limits...